scripts/configs.js

```diff
@@ -53,3 +53,11 @@ export const getConfigValue = async (configParam, environment) => {
   const configElements = JSON.parse(configJSON).data;
   return configElements.find((c) => c.key === configParam)?.value;
 };
+
+export const getCookie = (expectedCookieName) => {
+  const matches = document.cookie.match(
+    /* eslint-disable-next-line */
+    new RegExp(`(?:^|; )${expectedCookieName.replace(/([.$?*|{}()\[\]\\\/+^])/g, '\\$1')}=([^;]*)`),
+  );
+  return matches ? decodeURIComponent(matches[1]) : undefined;
+};
```

Comment:
Something like the following would have identical behavior but be more readable :)

```suggestion
export const getCookie = (cookieName) => {
  const cookies = document.cookie.split(';');
  for (const cookie of cookies) {
    const [name, value] = cookie.trim().split('=');
    if (name === cookieName) {
      return decodeURIComponent(value);
    }
  }
  return undefined;
};
```

Reply:
Make sense 👍 

I've did it slightly differently since ESLint don't like for loop 🙈 However, RegExp eliminated 👍 

> ESLint: iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.(no-restricted-syntax)